pkg/common/config_test.go

```diff
@@ -84,8 +84,8 @@ func TestLoadConfigWithContextConfig_FromCopiedTempFile(t *testing.T) {
 	assert.Equal(t, "http://localhost:8545", cfg.Context["devnet"].Chains["l2"].RPCURL)
 
 	// Fork blocks updated to v0.0.6 values
```

Comment:
nit: drop/update this comment if its not checking for v0.0.6 values anymore (swap v0.0.6 to latest)

Reply:
removed comment